### DISCUSSION TITLE
绳网并非完全无人之地

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#4719] [其他]
#### 最近，广告分子特别猖狂，不仅毫无顾忌的打广告，还涉及金钱交易，在推送上接连打出三个广告，甚至在issues上发了一个。。
![Screenshot_20240910_013333_com.microsoft.emmx.canary_edit_1732744089403330.jpg](https://github.com/user-attachments/assets/ca59ccb6-e8ed-4894-b1d4-74f56e8c45f4)
![Screenshot_20240910_080113_com.github.android_edit_1692842983649332.jpg](https://github.com/user-attachments/assets/414640b0-1772-4cb7-b23f-e60da2456180)

根据绳网的规则规定如下：

> ### 3.2 交易与广告 
> - **定义：**
> > - **商业推广信息：** 包括任何形式的商业推广性质的活动链接（如促销、优惠券、邀请码等）、二维码引导下载或买卖、个人联系方式（如QQ号、微信号等）用于私下交易，以及欺骗性诱导外链。
> >  - **交易规定：** 在社区内进行的交易活动必须遵循既定的流程，禁止私下交易；禁止涉及送号、卖号、抽奖送粉丝、送福袋、捐赠、录屏抽奖等活动，除非事先获得官方批准且活动完全公开透明。 
> > - **后果：** 违反上述规定的用户将面临警告、删帖等处罚。

在此提出警告，并且已将帖子删除。

SO，绳网在工作日确实没什么人管，但不代表就真无人咯，今天的公告就到此为止了，拜拜～

**Comment by lllliufeng** [comment]
智齿正义执行

**Comment by BJ-cmyk** [comment]
二维码涂一下啊哥
![1723465283508](https://github.com/user-attachments/assets/f3f3b016-fb73-4ee3-9dba-a5ce3b5c0590)

**Reply by nostalgiatan** [reply]
收到，已修正。

**Comment by tuneinnocent** [comment]
为什么有些帖子发的图不能直接在客户端看到呢？

**Reply by nostalgiatan** [reply]
大概率是网络问题
